_posts/2025-04-29-release-notes-3.3.6.md

```diff
@@ -0,0 +1,31 @@
+---
+category: release
+permalink: /news/3.3.6/
+title: "Scala 3.3.6 LTS is now available!"
+---
+Scala 3.3.6 LTS is now available!
+
+This patch release backports most of the improvements and bugfixes introduced in the Scala Next series up to the Scala 3.6.4 release.
```

Comment:
```suggestion
This patch release backports most of the bugfixes and some of the improvements introduced in the Scala Next series up to the Scala 3.6.4 release.
```

maybe? numerically maybe it's "most", but generally the most significant improvements aren't backported because they can't be

anyway, as you see fit

Reply:
Makes sense, thanks!